charts/zqd/templates/deployment.yaml

```diff
@@ -105,6 +113,30 @@ spec:
               name: spaces-volume
             - mountPath: /tmp
               name: tmp-volume                   
+            {{- if eq .Values.personality "root" }}
+            - mountPath: /creds/postgres
+              name: postgres-creds-volume
+            {{- end }}
+
+      {{- if eq .Values.personality "root" }}
```

Comment:
How do we handle the ordering for which of these containers does its work "first"? Does it matter if the migration happens before, after, or during startup of the zqd root process?

Reply:
The migration is an initContainer of the root process (i.e. "initdb"), so the root process won't start until after the migration has successfully exited.